cmd/kube-apiserver/app/server.go

```diff
@@ -511,7 +516,24 @@ func getRuntimeConfigValue(s *options.APIServer, apiKey string, defaultValue boo
 }
 
 // Parses the given runtime-config and formats it into map[string]ApiGroupVersionOverride
-func parseRuntimeConfig(s *options.APIServer) (map[string]genericapiserver.APIGroupVersionOverride, error) {
+func parseRuntimeConfig(s *options.APIServer) (genericapiserver.APIResourceEnablement, error) {
+	v1GroupVersionString := "api/v1"
+	v1GroupVersion := unversioned.GroupVersion{Group: api.GroupName, Version: "v1"}
```

Comment:
> hmm, config vs runtime dependency, I suppose

Yes.  When building `RESTStorage` directly, there's a strong argument for taking the explicit dependency.  There's less strength to it here and I know that @lavalamp doesn't want it long term.

All the special handling done in this method is nasty through.